docs/src/cncf-projects/external-secrets.md

```diff
@@ -0,0 +1,148 @@
+# External Secrets
+
+[External Secrets](https://external-secrets.io/latest/) is a CNCF Sandbox
+project, accepted in 2022 under the sponsorship of TAG Security.
+
+## About
+
+The **External Secrets Operator (ESO)** is a Kubernetes operator that enhances
+secret management by decoupling the storage of secrets from Kubernetes itself.
+It enables seamless synchronization between external secret management systems
+and native Kubernetes `Secret` resources.
+
+ESO supports a wide range of backends, including:
+
+- [HashiCorp Vault](https://www.vaultproject.io/)
+- [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/)
+- [Google Secret Manager](https://cloud.google.com/secret-manager)
+- [Azure Key Vault](https://azure.microsoft.com/en-us/services/key-vault/)
+- [IBM Cloud Secrets Manager](https://www.ibm.com/cloud/secrets-manager)
+
+…and many more. For a full and up-to-date list of supported providers, refer to
+the [official External Secrets documentation](https://external-secrets.io/latest/).
+
+## Integration with PostgreSQL and CloudNativePG
+
+When it comes to PostgreSQL databases, External Secrets integrates seamlessly
+with [CloudNativePG](https://cloudnative-pg.io/) in two major use cases:
+
+- **Automated password management:** ESO can handle the automatic generation
+  and rotation of database user passwords stored in Kubernetes `Secret`
+  resources, ensuring that applications running inside the cluster always have
+  access to up-to-date credentials.
+
+- **Cross-platform secret access:** It enables transparent synchronization of
+  those passwords with an external Key Management Service (KMS) via a
+  `SecretStore` resources. This allows applications and developers outside the
+  Kubernetes cluster—who may not have access to Kubernetes secrets—to retrieve
+  the database credentials directly from the external KMS.
+
+## Example: Automated Password Management with External Secrets
+
+Let’s walk through how to automatically rotate the password of the `app` user
+every 24 hours in the `cluster-example` Postgres cluster from the
+[quickstart guide](../quickstart.md#part-3-deploy-a-postgresql-cluster).
```

Comment:
While testing this I found out that it's not clear that the cluster must exists and be running first